JavaImportTool/src/javaImportTool/ASTConverter.java

```diff
@@ -654,17 +653,19 @@ public void processTypeParameters(List<TypeParameter> params) throws ConversionE
 		}
 	}
 	
-	public void processParameters(List<SingleVariableDeclaration> params) throws ConversionException
+	public void processParameters(List<VariableDeclaration> params) throws ConversionException
 	{
 		int i = 0;
-		for(SingleVariableDeclaration arg : params)
+		for(VariableDeclaration arg : params)
 		{
 			Node a = containers.peek().child("arguments").add(new Node(null,"FormalArgument", i++));
 			a.setSymbol(arg.getName().getIdentifier());
 			setModifiersAndAnnotations(a, Modifier.PUBLIC, null);
 			
-			Node type = typeExpression(arg.getType(), "typeExpression");
-			a.setChild("typeExpression", addExtraDimensions(type,arg.getExtraDimensions()));
+			if (arg instanceof SingleVariableDeclaration) {
```

Comment:
I guess that's fair. Our support is anyway incomplete for a bunch of things, so let's not make things complicated before we actually need it.